utilities/cli/container.py

```diff
@@ -439,10 +446,45 @@ def build(
         if full_build_args:
             cmd.extend(shlex.split(full_build_args))
 
-        cmd.extend(["-f", str(docker_file_path), "-t", img, str(HoloHubContainer.HOLOHUB_ROOT)])
+        cmd.extend(
+            [
+                "-f",
+                str(docker_file_path),
+                "-t",
+                img,
+                *(["-t", f"{img}-base"] if build_scripts else []),
+                str(HoloHubContainer.HOLOHUB_ROOT),
+            ]
+        )
 
         run_command(cmd, dry_run=self.dryrun)
 
+        if build_scripts:
```

Comment:
add comments above that of the flow of things

Reply:
Updated function header comment